resources/dictionary.txt

```diff
@@ -16,16 +16,16 @@ codebook
 CSER
 CSHL
 css
-CurrentAnVIL
```

Comment:
An option to keep these variable names from triggering the spell check is to use backticks, eg, ``` `CurrentAnVIL` ```.. this has the added benefit of formatting it like code, as well.